daemon/cmd/daemon_main.go

```diff
@@ -948,6 +948,24 @@ func initializeFlags() {
 	flags.Bool(option.HubbleSkipUnknownCGroupIDs, true, "Skip Hubble events with unknown cgroup ids")
 	option.BindEnv(Vp, option.HubbleSkipUnknownCGroupIDs)
 
+	flags.StringSlice(option.HubbleMonitorEvents, []string{},
+		fmt.Sprintf("Cilium monitor events for Hubble to observe: %s. By default, Hubble observes all monitor events.", strings.Join(
+			[]string{
+				monitorAPI.MessageTypeNameDrop,
+				monitorAPI.MessageTypeNameDebug,
+				monitorAPI.MessageTypeNameCapture,
+				monitorAPI.MessageTypeNameTrace,
+				monitorAPI.MessageTypeNameL7,
+				monitorAPI.MessageTypeNameAgent,
+				monitorAPI.MessageTypeNamePolicyVerdict,
+				monitorAPI.MessageTypeNameRecCapture,
+				monitorAPI.MessageTypeNameTraceSock,
```

Comment:
What about adding something like this in `pkg/monitor/api/types.go`:

```
var MessageTypeFilterAll = MessageTypeFilter{
	MessageTypeDrop,
	MessageTypeDebug,
	MessageTypeCapture,
	MessageTypeTrace,
	MessageTypeAccessLog,
	MessageTypeAgent,
	MessageTypePolicyVerdict,
	MessageTypeRecCapture,
	MessageTypeTraceSock,
}
```

and then change this to:

```
fmt.Sprintf(
    "Cilium monitor events for Hubble to observe: [%s]. By default, Hubble observes all monitor events.",
    monitorAPI.MessageTypeFilterAll.String(),
),
```

This seems cleaner to me, and in case of new filters it reduces the chances of forgetting to update the list in the message.

Reply:
Done